software/programming_instructions.md

```diff
@@ -91,7 +92,18 @@ Now you have access to the inputs and outputs using easy methods, which you can
   
   ![image](https://user-images.githubusercontent.com/79809962/151054018-0f495bb5-067e-4cd6-9640-c38e44a216de.png)  
   
-  
+### Install the contrib scripts and setup the menu
+
+1. Use the exact same process as for the ssd1306 and europi libraries to install the europi-contrib library:
+   1. In Thony, click Tools -> Manage Packages to open the package manager.
+   2. Type 'europi' into the search box and click 'Search on PyPi'
+   3. Click the result named 'micropython-europi-contrib'.
+   4. Click 'Install'.
+   5. You will now see the new directory 'contrib' inside the 'lib' folder, containing the contrib scripts.
+2. The menu program is called ``bootloader.py`` and can be found in the lib directory on the pico or in the [``software/firmware``](/software/firmware/bootloader.py) directory on github. Copy this file as your ``main.py`` as described in 
```

Comment:
I think this is outdated, there now seem to be two separate files, ``bootloader.py`` which should not be altered, and ``menu.py`` which should be.
This segment (I think) should read:
"The menu program is called ``menu.py`` and can be found in the lib directory on the pico or in the [``software/contrib``](/software/contrib/menu.py) directory on GitHub."